systests/databinding/pom.xml

```diff
@@ -165,7 +165,7 @@
             <plugin>
                 <groupId>org.jibx</groupId>
                 <artifactId>maven-jibx-plugin</artifactId>
-                <version>${cxf.jibx.version}</version>
+                <version>1.3.1</version>
```

Comment:
Why are you removing a property version and putting a hard coded version number in there?